chatwoot/chatwoot
# fix: Auto reply on email inbox

## Description

This change allows the user to configure auto-reply on email inbox. 
User will receive an auto-reply(Greeting) message for every new conversation.

Fixes #3713

## Type of change


## How Has This Been Tested?

1. Setup an email inbox
2. Configure Auto-reply
3. Send a mail to email inbox configured.
4. New conversation should be created with incoming email content. 
5. An auto-reply message also should be visible in the same conversation.
6. Verify the auto-reply message in the sender email inbox.


## Checklist:
